backend/ssm/ssm.go

```diff
@@ -6,20 +6,23 @@ import (
 
 	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
+
 	"github.com/heetch/confita/backend"
 )
 
-type Backend struct {
+type ssmBackend struct {
 	client  ssmiface.SSMAPI
 	ssmPath string
 	cache   map[string][]byte
 }
 
-func NewBackend(ssm ssmiface.SSMAPI, path string) *Backend {
-	return &Backend{client: ssm, ssmPath: path}
+// NewBackend returns a backend instance that uses the given SSMAPI implementation
+// to retrieve keys from the parameter store at the given path.
+func NewBackend(ssm ssmiface.SSMAPI, path string) backend.Backend {
+	return &ssmBackend{client: ssm, ssmPath: path}
 }
 
```

Comment:
Public methods should have documentation strings.

Reply:
done